### PR TITLE
Dashboard backend

### PR DESCRIPTION
**Pull request by leonprou**
## Proposed changes

Backend for the dashboard:
- Store Transfer events to db
- make an API route for aggregation of events

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: server/src/models/Event.js [hunk 1]
```diff
@@ -1,12 +1,21 @@
+const pickBy = require('lodash').pickBy
+const mongoose = require('mongoose')
 
-module.exports = (mongoose) => {
-  mongoose = mongoose || require('mongoose')
+const manipulate = (data) => {
+  // delete double entries in returnValues
+  const returnValues = pickBy(data.returnValues, (value, key) => isNaN(key))
```
Comment (LiorRabin) [inline server/src/models/Event.js]: cool :)

File: server/src/services/agenda.js [hunk 2]
```diff
@@ -31,6 +35,10 @@ async function start () {
   await agenda.schedule('in 1 minute', 'processPastMarketOpenEvents')
   await agenda.every('10 minutes', 'processPastMarketOpenEvents', null, {skipImmediate: true})
 
+  // run the task every day in 09:00 UTC (11:00 Israel)
+  await agenda.every('00 9 * * *', 'processPastTransferEvents')
+  await agenda.schedule('in 3 minute', 'processPastTransferEvents')
```
Comment (LiorRabin) [inline server/src/services/agenda.js]: once when the app starts and after that every day at 9 right?

File: server/src/utils/events/handlers.js [hunk 3]
```diff
@@ -22,9 +22,16 @@ const handleMarketOpenEvent = async (event) => {
   return communityUtils.openMarket(mmAddress)
 }
 
+const handleTransferEvent = (event) => {
```
Comment (LiorRabin) [inline server/src/utils/events/handlers.js]: I'm not sure I understand why you need this function - just logging?

File: server/src/services/agenda.js (hunk 2 above)
Comment (leonprou) [reply server/src/services/agenda.js]: Yeah exactly

File: server/src/utils/events/handlers.js (hunk 3 above)
Comment (leonprou) [reply server/src/utils/events/handlers.js]: Yes kinda, for other types of event I change the db tables on receive (see the [whole file](https://github.com/ColuLocalNetwork/CLN-community-app/blob/0277b6e9fa82d575df22a692ec8fb555a5cfe013/server/src/utils/events/handlers.js)). But the `Transfer` event I just store them for now.

**Pull request merged**